docs/conf.py

```diff
@@ -22,6 +22,8 @@
         'resolwe.flow',
         'resolwe_bio',
     ),
+    FLOW_MANAGER={'REDIS_PREFIX': 'xxx'},
+    FLOW_EXECUTOR={},
```

Comment:
@jberci please make sure Resolwe will run even if these settings will be missing.

Reply:
This should work now, I think, so you don't need these lines anymore.